### DISCUSSION TITLE
How to setup Wiki.Js in my cPanel managed server?

### DISCUSSION THREAD

**Discussion by ashwinbalaji0811** [requarks/wiki#3403] [Help / Questions]
I have a shared hosting server running and would like to use Wiki.Js for my personal Wiki. I am not an expert in nodejs or docker or any other technologies. Please help me with the steps in which I can set up Wiki.Js in my own website.

**Comment by Smankusors** [comment]
Is your shared hosting does have something like Plesk? Or is your cPanel have menu for node.js application? Or maybe can you do SSH and check if `node` installed there?

If not.... I'm afraid you can't do this on your shared hosting.

**Comment by ashwinbalaji0811** [comment]
![image](https://user-images.githubusercontent.com/28635635/86992563-62e04280-c1bf-11ea-828c-c8fe6cbce06b.png)
I have node.js app installer in my cPanel.

**Comment by Smankusors** [comment]
Ah yes, you can use that "Setup Node.js App". Is Node.js 10.12 or later available at that menu?

**Comment by ashwinbalaji0811** [comment]
But can you link me to proper setup steps which I can follow?

I read in the documentation that future major release will [drop support](https://docs.requarks.io/install/requirements) for all database except Postgresql. So, should I set it up with PostgreSQL so that I will not face trouble while I update it to 3.x later in future?

**Comment by Smankusors** [comment]
> But can you link me to proper setup steps which I can follow?
> 
> I read in the documentation that future major release will [drop support](https://docs.requarks.io/install/requirements) for all database except Postgresql. So, should I set it up with PostgreSQL so that I will not face trouble while I update it to 3.x later in future?

unfortunately I do not have any setup steps that specific for Wiki.js. This should be added to Wiki.js installation documentation though... But any generic "cpanel node setup" Google search should help you setup Wiki.js in shared hosting. #1866 could help you if you confused with what input you should entered in the cPanel.

If you still confused, just ask here again. But, me never tried this btw. 😅

regarding database support, ask @NGPixel 😅

**Comment by ashwinbalaji0811** [comment]
while installing, which is the application starter file I have to link it in the setup??

is it `config.yml` or the ones in `server\index.js` or `server\master.js` or `server\setup.js`

**Comment by Smankusors** [comment]
> while installing, which is the application starter file I have to link it in the setup??
> 
> is it `config.yml` or the ones in `server\index.js` or `server\master.js` or `server\setup.js`

it's `server\index.js` according to #1866.
